source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -78,7 +82,32 @@ Settings
 
        |
        | **Default**: ``""``
-       | **Accepted Values**: ``""`` or ``"default"`` or ``"updateLookup"``
+       | **Accepted Values**: ``""``, ``"updateLookup"``, ``"whenAvailable"``, or ``"required"``
+       
+   * - | **change.stream.full.document.before.change**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Configures the document pre-image your change stream returns on update
+         operations. The pre-image is not available in source records
+         published while copying existing data, and the pre-image configuration
+         has no effect on copying.
+         
+       .. note::
+       
+          To learn about how to configure a collection to enable
```

Comment:
Issue:

Remove "about" and change it to "To learn how to configure..." It's a useless word in this sentence.